lib/mix/lib/mix/dep/converger.ex

```diff
@@ -48,6 +48,9 @@ defmodule Mix.Dep.Converger do
   """
   def converge(acc, lock, opts, callback) do
     {deps, acc, lock} = all(acc, lock, opts, callback)
+    remote = Mix.RemoteConverger.get
+    # TODO: Remove function_exported? check before releasing 1.4
```

Comment:
The best would be to open an issue and add a milestone so we don't forget about this. I personally don't check TODOs before a release.